pkgs/development/python-modules/Kconfiglib/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub }:
+buildPythonPackage rec {
+  pname = "Kconfiglib";
+  version = "10.36.0";
+
+  src = fetchFromGitHub {
+    owner = "ulfalizer";
+    repo = "Kconfiglib";
+    rev = "v${version}";
+    sha256 = "0dzdak70ss56n68qssxi9fi9b55y36wj3xiph5y7dw7dbjqax9fi";
+  };
+
+  doCheck = false;
```

Comment:
Why?

Reply:
tests expect to be run from an out of tree folder. Then I got scared it would lead me to a whole lot of trouble with kernel interactions. I might have a look at this in a future update.